src/wfs/src/main/java/org/geoserver/wfs/WFSGetFeatureOutputFormat.java

```diff
@@ -212,8 +212,10 @@ public void write(Object value, OutputStream output, Operation operation)
         return null;
     }
 
-    protected int getNumDecimals(List featureCollections, GeoServer geoServer, Catalog catalog) {
+    protected int[] getNumDecimalsEncodeMeasures(
```

Comment:
Ugh... I'd go for two separate methods with clearer names, if the copied code is too much, maybe you can pass in lambdas? It's true that mechanically one has to perform a similar operation, but decimals and measures are conceptually separate and actually orthogonal.

Reply:
_It's true that mechanically one has to perform a similar operation, but decimals and measures are conceptually separate and actually orthogonal._

It was not about what the meaning of the values but about the operation which is the same: **getting meta-information from the feature type info**.

That say, I agree with you when you say _I'd go for two separate methods with clearer names_, I will do this.